v1.0_metadata.xml

```diff
@@ -2503,7 +2503,7 @@
         <NavigationProperty Name="directReports" Type="Collection(graph.directoryObject)" />
         <NavigationProperty Name="licenseDetails" Type="Collection(graph.licenseDetails)" ContainsTarget="true" />
         <NavigationProperty Name="manager" Type="graph.directoryObject" />
-        <NavigationProperty Name="memberOf" Type="Collection(graph.directoryObject)" />
+        <NavigationProperty Name="memberOf" Type="Collection(graph.group)" />
```

Comment:
this should stay as it was. A user can be a member of a group, but also of a role. And this collection can return any of those, this is why it's using the parent type. If you want the collection to be filtered as groups (and down casted), you can append `/microsoft.graph.group` to your request.